www/_plugins/lang_polyfill.rb

```diff
@@ -0,0 +1,33 @@
+# encoding: utf-8
+
+
+
+module Jekyll
+	class LangPolyfill < Generator
+		priority :high
+
+		def generate(site)
+			fakes = site.pages.map do |p|
+				next if p.data['lang'] != 'de'
+				next if File.exist?("en/#{p.name}")
+
+				copy = p.dup
+				copy.data['lang'] = 'en'
+				copy.process("en/#{copy.name}")
+				copy.data['title'] << " (untranslated, sorry)"
+
+				notice = <<~NOTICE
+				# Translation missing
```

Comment:
It might be sensible to put this in a `<div>` with a class, so that we can highlight it specifically and make it stand out as an info-box (As done [here](https://jekyllrb.com/docs/usage/) like [so](https://raw.githubusercontent.com/jekyll/jekyll/master/docs/_docs/usage.md))